package.json

```diff
@@ -209,8 +209,7 @@
       "eslint --ext js,ts,tsx --fix"
     ],
     "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
-      "prettier --write",
-      "git add"
+      "prettier --write"
```

Comment:
What was this change for? Try to limit your changes to a tight scope in the future.

Reply:
Ok. Sorry.

That was a fix for new version of `lint-staged`, where `git add` is not required because the lib runs it for you automatically